src/views/AddTenant/addTenant.js

```diff
@@ -5,7 +5,6 @@ import * as Yup from "yup";
 import * as axios from "axios";
 import UserContext from "../../UserContext";
 import Button from "../../components/Button";
-import { AddProperty } from '../addProperty/addProperty';
```

Comment:
This needs to be put back in since it's being used in the render function below to create a new property